src/api/getTickerWidgetData.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+import { Interval } from 'components/Widget/Widget.constants';
+import type { IWidgetTicker } from 'components/Widget/Widget.types';
+
+export default async function getTickerWidgetData(
+  pairId: string,
+  interval: Interval,
+  type: 'current' | 'previous' = 'current',
+): Promise<IWidgetTicker> {
+  try {
+    const response = await axios.get(
+      `https://api.investing.com/api/financialdata/${pairId}/historical/chart/?interval=${interval}&pointscount=60`,
+    );
+
+    const dataIndex =
+      type === 'current'
+        ? response.data.data.length - 1
```

Comment:
계속 반복해서 사용한다면, 변수로 할당해 주는것은 어떨까요?
```suggestion
const responseData = response.data.data
```